homeassistant/components/sonos/media_browser.py

```diff
@@ -508,12 +526,16 @@ def get_media_type(item: DidlObject) -> str:
     return SONOS_TYPES_MAPPING.get(item.item_id.split("/")[0], item.item_class)
 
 
-def can_play(item: DidlObject) -> bool:
+def can_play(item_class: str, item_id: str | None = None) -> bool:
     """Test if playable.
 
     Used by async_browse_media.
     """
-    return SONOS_TO_MEDIA_TYPES.get(item) in PLAYABLE_MEDIA_TYPES
+    # Folders are playable once we reach the folder level.
+    # Format is S://server_address/share/folder/...
+    if item_id and item_id.startswith("S:") and item_class == "object.container":
+        return item_id.count("/") > 3
```

Comment:
I count 5 `/` in the example at L535, not sure if 3 is correct then

Reply:
3 is correct; but the comment did have 5 slashes in it; when there should be 4. Update the comment and change the comparison to be >= 4

Reply:
apolgoies for the delay, was on travel